packages/tsconfig/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "NL Design System",
+  "docs": "https://github.com/nl-design-system/basis/blob/main/packages/tsconfig#readme",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": false,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "allowArbitraryExtensions": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react-jsx",
+    "lib": ["ESNext"],
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitOverride": true,
+    "noImplicitReturns": true,
+    "noPropertyAccessFromIndexSignature": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "paths": {},
+    "pretty": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "ES2023"
+  }
+}
```

Comment:
Ik stel voor de `tsconfig.json` files in src/ te ztten, zodat we daar ook bijvoorbeeld `react.tsconfig.json` enzo kunnen zetten.

Reply:
Mijn idee was om daar `react/tsconfig.json` etc van te maken. Dat leek wat mij betreft nog het meest op de bases die `@tsconfig` zelf heeft.

Reply:
De base configs gebruiken een script om packages te genereren maar in die packages staan de JSON configuraties allemaal in de root en zijn er geen `"exports". Het enige verschil is dat die bases 1 config per package hebben maar ik denkt dat dat prima op te lossen is met een directory per config waarin dan gewoon netjes een `tsconfig.json` kan zitten.